Specs/Scene/Points3DModel3DTileContentProviderSpec.js

```diff
@@ -0,0 +1,172 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/Points3DTileContentProvider',
+        'Core/Cartesian3',
+        'Core/defaultValue',
+        'Core/HeadingPitchRange',
+        'Core/OrientedBoundingBox',
+        'Scene/Cesium3DTileContentState',
+        'Scene/Cesium3DTileset',
+        'Specs/createScene',
+        'Specs/pollToPromise'
+    ], function(
+        Points3DTileContentProvider,
+        Cartesian3,
+        defaultValue,
+        HeadingPitchRange,
+        OrientedBoundingBox,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        createScene,
+        pollToPromise) {
+    "use strict";
+
+    var scene;
+    var centerLongitude = -1.31995;
+    var centerLatitude = 0.69871;
+
+    var pointsUrl = './Data/Cesium3DTiles/Points/Points';
+
+    beforeAll(function() {
+        scene = createScene();
+        scene.frameState.scene3DOnly = true;
```

Comment:
Instead of hacking this in afterwards, it would be better to pass this through with `createScene`.

Also, can you put a comment in this test about this since is, hopefully, only a temporary limitation.